tfac/PCA_functions.py

```diff
@@ -0,0 +1,24 @@
+"""Contains function for performing PCA on OHSU data"""
+from dataHelpers import Tensor_LINCS_MEMA
+from sklearn.preprocessing import StandardScaler
+from sklearn.decomposition import PCA
+import matplotlib.pyplot as plt
+import numpy as np
+
+
+def R2X_plot(datafile):
+  R2X_list = []
+  data = StandardScaler().Tensor_LINCS_MEMA(data)
+  for x in range(2, 21):
+      PCA_model = PCA(n_components=x)
+      PCA_model.fit(data)
+      R2X_list.append(np.sum(PCA_model.explained_variance_ratio_) * 100)
+  plt.plot(range(2, 21), R2X_list)
```

Comment:
you don't need to write an R2X function. You can use the one from tensorpack (take a look at figure1 to see how to use it.) I believe we don't need an R2X plot. Just run it for yourself to decide how many components you want. Although, you can decide from the reduction plot, too (figure1e).